packages/beacon-node/src/chain/blocks/blockInput/types.ts

```diff
@@ -0,0 +1,165 @@
+import {ForkName} from "@lodestar/params";
+import {ColumnIndex, RootHex, SignedBeaconBlock, Slot, deneb, fulu} from "@lodestar/types";
+
+export type CustodyConfig = {
```

Comment:
We should add a TODO here so we don't forget to update this to the correct object when we rebase `peerDAS` branch

```suggestion
// TODO(peerDAS): This needs to be updated to the actual `CustodyConfig` object
export type CustodyConfig = {
```